presets/imcs/cluster/CMakePresets.json

```
@@ -25,6 +37,19 @@
         "CMAKE_BUILD_TYPE": "RELEASE",
         "FOUR_C_TRILINOS_ROOT": "$env{TRILINOS_ROOT}"
       }
+    },
+    {
+      "name": "imcs_charon_debug",
+      "displayName": "Debug build for charon at IMCS",
+      "inherits": [
+        ".imcs_cluster"
+      ],
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "DEBUG",
+        "FOUR_C_ENABLE_ADDRESS_SANITIZER": "ON",
```

This combination is likely going to run incredibly slowly. I think the ASAN docs strongly recommend always using some optimization.

How about:

```suggestion
        "CMAKE_BUILD_TYPE": "RELWITHDEBINFO",
        "FOUR_C_ENABLE_ADDRESS_SANITIZER": "ON",
```